examples/gke-security-best-practices/network.tf

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# create a dedicate network for our cluster
+resource "google_compute_network" "cluster" {
```

Comment:
Creating a network just for the cluster is not really a best practice - Shared VPC is our best practice.

Reply:
agreed, will update to reflect